.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/cicognara-rails
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+    docker:
+      - image: circleci/ruby:2.3-node-browsers
+        environment:
+          RAILS_ENV: test
+          CICOGNARA_DB_HOST: localhost
+          CICOGNARAP_DB_USERNAME: cicognara
+    steps:
+      - checkout
+      - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+      # Dependencies
+      # Restore the dependency cache
+      - type: cache-restore
+        name: Restore bundle cache
+        key: cicognara-rails-{{ checksum "Gemfile.lock" }}
+
+      # - run: sudo update-alternatives --set java /usr/lib/jvm/jdk1.8.0/bin/java
```

Comment:
Can you get rid of these commented lines?

Reply:
yes, I was still working on that. sorry.

Reply:
Do you want the to run the rake test_server separately and include the jetty_wait script?

Reply:
It'd be a little faster, but I think this is fine the way it is.

Reply:
ok